tembo-operator/src/stacks/templates/api.yaml

```diff
@@ -1,7 +1,10 @@
 name: API
 description: Tembo Stack with REST and graphQL interfaces.
-repository: "quay.io/tembo"
-image: "standard-cnpg:15.3.0-1-3d9a853"
+organization: tembo
+images:
+  14: "standard-cnpg:pg14-f1721c2"
+  15: "standard-cnpg:15.3.0-1-3d9a853"
```

Comment:
The PG 15 image name needs to be like the others.  We now publish it as `*-cnpg:pg<pg version>-<git sha>`